data/json/dreams.json

```diff
@@ -545,6 +579,8 @@
     "type": "dream",
     "messages": [
       "You dream of the hunt, the feed, and your pack.",
+      "You lead the pack on a glorious hunt, knowing that they will all have your back as you have theirs!",
+      "You ride a black horse.  It is the end of days.  You are a salesman; one of a pack of four.  You sell only hunger, and you go to every door.",
```

Comment:
What does the Abrahamic eschaton scenario have to do with mutating into a partial-wolf-thing?

Reply:
"The wolf at the door" = famine

But I'll remove it